apps/users/emails.py

```diff
@@ -47,7 +48,12 @@ def get_attachments(self):
         organisation = self.get_organisation()
         if organisation and organisation.logo:
             f = open(organisation.logo.path, "rb")
-            logo = MIMEImage(f.read())
+            try:
+                logo = MIMEImage(f.read())
+            except TypeError:
+                f.seek(0)
+                filetype = magic.from_buffer(f.read(), mime=True)
+                logo = MIMEImage(f.read(), _subtype=filetype)
```

Comment:
here I would check `if logo:` and then add it to the attachements, else I would send a notification/log error. And IMO attachements should be returned eitherway, even if the logo didn't make it, otherwise emails gets lost.

Reply:
maybe better to use magic in the try block, and in the exception we send an email error and log?

Reply:
I just checked and magic doesn't throw an exception, if it's garbage data it will return `application/octet-stream` as mime type

Reply:
but I can move it in it's on try if you want